src/stories/Button/PageButton.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { action as fn } from '@storybook/addon-actions';
+import { colors, size } from '../../styles/Theme';
+import PageButton from '../../components/common/Button/PageButton/PageButton';
+
+const meta = {
+  title: 'Button/PageButton',
+  component: PageButton,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+
+  argTypes: {
+    color: {
+      control: 'color',
+      description: '페이지 버튼의 색상',
+      defaultValue: colors.PRIMARY,
+    },
+    text: {
+      control: 'text',
+      description: '페이지 버튼의 텍스트',
+      defaultValue: '버튼입니다.',
+    },
+    onClick: { action: 'clicked', description: '버튼 클릭 이벤트' },
+  },
+  args: {
+    onClick: fn('Button clicked'),
+  },
+} satisfies Meta<typeof PageButton>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    size: size.SIZE_008,
+    color: colors.WHITE,
+    onClick: () => {},
+  },
+};
```

Comment:
로컬에서 스토리북을 실행시켜봤는데, 스토리북이 뜨지 않네요,,!
 이 props 들을 
```suggestion
export const Default: Story = {
  args: {
    color: colors.WHITE,
    text: '1',
    isActive: false,
    onClick: () => {},
  },
};

```

이렇게 수정해야할 것 같아요!

Reply:
수정했어요!!